content/doc/book/blueocean/activity.adoc

```diff
@@ -13,37 +13,89 @@ wip: true
 
 = Activity View
 
-The Blue Ocean Activity Viewer shows the all activity related to one
+The Blue Ocean Activity View shows the all activity related to one
```

Comment:
I would use the the terms `Pipeline Activity tab`, `Pipeline Branches tab` and `Pipeline Pull Requests tab`